disconnected/installing.adoc

```diff
@@ -0,0 +1,113 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-disconnected-environments"]
+= Installing a cluster in a disconnected environment
+include::_attributes/common-attributes.adoc[]
+:context: installing-disconnected-environments
+
+toc::[]
+
+You can install an {product-title} cluster in a disconnected environment, choosing the installation method and infrastructure that best suits your requirements.
+This includes installing {product-title} on either on-premise hardware or on a cloud hosting service such as Amazon Web Services (AWS).
+
+The following sections outline all of the supported methods for installing a cluster in a disconnected environment.
+
+[NOTE]
+====
+In order to learn about other requirements for installing a cluster using a particular method, be sure to review other content in the procedure's respective section of the documentation.
```

Comment:
```suggestion
In order to learn about other requirements for installing a cluster using a particular method, be sure to review other content in the procedure's respective section of the documentation. For example, if you install a cluster on {aws-short} with installer-provisioned infrastructure, see the xref:../installing/installing_aws/installing-aws-account.adoc#installing-aws-account[Configuring an AWS account] and xref:../installing/installing_aws/ipi/ipi-aws-preparing-to-install.adoc#ipi-aws-preparing-to-install[Preparing to install a cluster on {aws-short}] pages.
```

is this what you meant by this note? if so, I think an example would be useful. and please do check my syntax because I wrote it in a github comment box, not vscode :P

Reply:
Yeah that is what I meant, and sure I can add this example.